### PR TITLE
Added publish dependencies for ios

### PR DESCRIPTION
**Pull request by boztopuz**

### CODE REVIEW

File: appcircle_publish_send_to_testflight/1.2.1/component.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+platform: iOS
+purpose: Publish
+marketPlace: "AppleAppStore"
+buildPlatform:
+displayName: "Send to TestFlight"
+description: "Upload binary for TestFlight publishing"
+docsUrl: https://docs.appcircle.io/publish-integrations/ios-publish-integrations/sent-to-testflight
+requiredComponents: 
+precedingComponents: "appcircle_publish_appstore_app_information"
```
Comment (ftmzhrsckldr) [inline appcircle_publish_send_to_testflight/1.2.1/component.yaml:9]: Why we need to add this step after the appcircle_publish_appstore_app_information step?

File: appcircle_publish_testflight_status_check/1.1.1/component.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+platform: iOS
+purpose: Publish
+structureType: CustomUI
+buildPlatform:
+displayName: "Get Approval from TestFlight"
+description: "Get approval for the publish from TestFlight"
+docsUrl: https://docs.appcircle.io/publish-integrations/ios-publish-integrations/approval-test-flight
+requiredComponents: "appcircle_publish_send_to_testflight"
+precedingComponents: "appcircle_publish_appstore_app_information"
```
Comment (ftmzhrsckldr) [inline appcircle_publish_testflight_status_check/1.1.1/component.yaml:9]: Why we need to add this step after the appcircle_publish_appstore_app_information step?

**Review by ftmzhrsckldr — CHANGES_REQUESTED** [review]

File: appcircle_publish_send_to_appstore/1.2.1/component.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+platform: iOS
+purpose: Publish
+marketPlace: "AppleAppStore"
+buildPlatform:
+displayName: "Send to App Store"
+description: "Upload binary for App Store publishing"
+docsUrl: https://docs.appcircle.io/publish-integrations/ios-publish-integrations/send-to-app-store
+requiredComponents: 
+precedingComponents: "appcircle_publish_send_to_testflight,appcircle_publish_testflight_status_check,appcircle_publish_appstore_app_information"
```
[the remark this answers is not in the document]
Comment (boztopuz) [reply appcircle_publish_send_to_appstore/1.2.1/component.yaml:9]: Actually, email approval step is common step, I think it should be free in the flow.

File: appcircle_publish_appstore_status_check/1.0.1/component.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+platform: iOS
+purpose: Publish
+buildPlatform:
+displayName: "Check App Store Release Status"
+description: "Checks statuses of the app on App Store"
+docsUrl: https://github.com/appcircleio/appcircle-publish-appstore-status-check
```
[the remark this answers is not in the document]
Comment (boztopuz) [reply appcircle_publish_appstore_status_check/1.0.1/component.yaml:6]: Yes I know, but the doc is missing. I fix the doc and then I changed the doc link.

File: appcircle_publish_send_to_appstore/1.2.1/component.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+buildPlatform:
+displayName: "Send to App Store"
+description: "Upload binary for App Store publishing"
+docsUrl: https://docs.appcircle.io/publish-integrations/ios-publish-integrations/send-to-app-store
+requiredComponents: 
+precedingComponents: "appcircle_publish_send_to_testflight,appcircle_publish_testflight_status_check"
+followingComponents: 
+webUrl: https://github.com/appcircleio/appcircle-publish-send-to-appstore
+repoUrl: https://github.com/appcircleio/appcircle-publish-send-to-appstore.git
+commit: 11e6ef4
```
Comment (ftmzhrsckldr) [inline appcircle_publish_send_to_appstore/1.2.1/component.yaml:13]: Commit id not same the last commit ID, is that any reason?

File: appcircle_custom_script/1.0.2/component.yaml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+componentType: appcircle_custom_script
+displayName: Custom Script
+description: "Carry out any custom operation in the build machine with a compatible script."
+docsUrl: https://docs.appcircle.io/workflows/common-workflow-steps/custom-script
+requiredComponents: 
+precedingComponents: 
+followingComponents: 
+webUrl: https://github.com/appcircleio/appcircle-custom-script
+repoUrl: https://github.com/appcircleio/appcircle-custom-script.git
+commit: c163e15
```
Comment (ftmzhrsckldr) [inline appcircle_custom_script/1.0.2/component.yaml:13]: Commit id not same the last commit ID, is that any reason?

**Pull request merged**